src/app/dialog/dialog.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { Product } from '../interfaces/product';
+import { LocalStorageService } from '../services/local-storage.service';
+import { Dialog } from '../constants/Dialog';
+
+@Component({
+  selector: 'app-dialog',
+  templateUrl: './dialog.component.html',
+  styleUrls: ['./dialog.component.scss'],
+})
+export class DialogComponent implements OnInit {
+  public numberOfproducts = Dialog.MIN_NUMBER_OF_PRODUCTS;
```

Comment:
For defalut value we should use model instead of constants. According to copying of values